test/Microsoft.ML.Predictor.Tests/TestAutoInference.cs

```diff
@@ -362,6 +362,73 @@ public void EntryPointPipelineSweepRoles()
             Assert.True(rows.All(r => r.TrainingMetricValue > 0.1));
         }
 
+        [Fact]
+        [TestCategory("EntryPoints")]
+        public void EntryPointPipelineSweepMultiClass()
+        {
+            // Get datasets
+            var pathData = GetDataPath("adult.train");
+            var pathDataTest = GetDataPath("adult.test");
+            const int numOfSampleRows = 100;
+            const string schema =
+                "sep=, col=age:R4:0 col=workclass:TX:1 col=fnlwgt:R4:2 col=education:TX:3 col=education_num:R4:4 col=marital_status:TX:5 col=occupation:TX:6 " +
+                "col=relationship:TX:7 col=ethnicity:TX:8 col=sex:TX:9 col=Features:R4:10-12 col=native_country:TX:13 col=IsOver50K_:R4:14 header=+";
+            var inputFileTrain = new SimpleFileHandle(Env, pathData, false, false);
+#pragma warning disable 0618
```

Comment:
disables warning generated because of using ImportText (obsolete)   instead of Textloader.  

Perhaps in a future PR, we can update all the unit tests to use TextLoader. I can create a separate bug to track that.

---
In reply to: [202746784](https://github.com/dotnet/machinelearning/pull/539#discussion_r202746784) [](ancestors = 202746784)